introduction.md

```diff
@@ -0,0 +1 @@
+I'm Steen a git sempai practicing in this dojo in order to become a real git shihan!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.